ansible/group_vars/all

```diff
@@ -5,4 +5,5 @@ histomicstk_version: master
 large_image_version: master
 dsa_version: master
 girder_worker_version: master
+itk_version: master
```

Comment:
Probably. We can do the same for others also, if people have suggested tags please let me know.